resources/remote-control.txt

```diff
@@ -23,6 +23,8 @@ Supported commands:
     Get the value of the gain setting with the name <gain_name>
  L <gain_name>_GAIN <value>
     Set the value of the gain setting with the name <gain_name> to <value>
+ p
+    Get the RDS PI code (in hexadecimal). Returns 0000 if not applicable.
```

Comment:
Gqrx's remote control protocol is meant to be compatible with [rigctld's](https://www.systutorials.com/docs/linux/man/8-rigctld/). I see it uses `p` to get a parameter (which probably makes sense here) but also has a parameter name. Perhaps this should be `p RDS_PI`? That would also leave room for future expansion. (For example, it might be useful to fetch other RDS data.)